README.md

```diff
@@ -11,13 +11,62 @@ This project uses deep learning computer vision to label images taken by motion-
 - [Uptake.org Autofocus Case Study](https://www.uptake.org/impact/special-projects)
 - [Machine Learning Meets Wildlife Conservation](https://www.lpzoo.org/blog/machine-learning-meets-wildlife-conservation)
 
+## Getting the App
+
+If you just want to get labels for your images, you can use the following steps to run a service that passes images through a trained model.
+
+1. Make sure [Docker](https://www.docker.com/get-started) is installed and running.
+2. Run `docker pull gsganden/autofocus` to download the app image. (Note that it takes up about 4GB of disk space.)
+3. Run `docker run -p 8000:8000 gsganden/autofocus_serve` to start the app.
+4. Make POST requests against the app to get predictions. For instance, with the base of this repo as the working directory you can send the image `fawn.jpeg` to the app with `curl -F "file=@./gallery/fawn.jpeg" -X POST http://0.0.0.0:8000/predict`. Or you can create a zip file of the `gallery` directory and send it to the app with `curl -F "file=@gallery.zip" -X POST http://0.0.0.0:8000/predict_zip`. See `autofocus/predict/example_post.py` or `autofocus/predict/example_post.R` for example scripts that make requests using Python and R, respectively.
+
+For a single image, the app will respond with a JSON object that indicates the model's probability that the image contains an animal in each of the categories that it has been trained on. For instance, it might give the following response for an image containing raccoons:
+
+```json
+{
+  "beaver": 7.996849172335282e-16,
+  "bird": 6.235780460883689e-07,
+  "cat": 9.127776934292342e-07,
+  "chipmunk": 4.231552441780195e-09,
+  "coyote": 2.1184381694183685e-05,
+  "deer": 3.6601684314518934e-06,
+  "dog": 1.4745426142326323e-06,
+  "empty": 0.0026697132270783186,
+  "fox": 2.7905798602890358e-14,
+  "human": 1.064212392520858e-05,
+  "mink": 2.7622977689933936e-13,
+  "mouse": 4.847318102463305e-09,
+  "muskrat": 6.164089044252078e-16,
+  "opossum": 9.763967682374641e-05,
+  "rabbit": 2.873173616535496e-05,
+  "raccoon": 0.9986177682876587,
+  "rat": 4.3888848111350853e-10,
+  "skunk": 4.078452775502228e-07,
+  "squirrel": 1.2888597211713204e-06,
+  "unknown": 0.0004612557531800121,
+  "woodchuck": 1.2980818033154779e-14
+}
+```
+
+The `predict_zip` endpoint returns a JSON object mapping file paths to model probabilities formatted as above.
```

Comment:
it's common to document endpoints with a leading `/`, so `/predict_zip`. This visually distinguishes them from functions or classes in your code

Reply:
Fixed, thanks!